src/py/flwr/client/app.py

```diff
@@ -204,6 +205,9 @@ def _start_client_internal(
         The IPv4 or IPv6 address of the server. If the Flower
         server runs on the same machine on port 8080, then `server_address`
         would be `"[::]:8080"`.
+    node_config: Dict[str, str]
+        The configuration of the node, it must contain at least a `num-partitions`
+        and a `partition-id` key.
```

Comment:
This is not true - there is not requirement for `node_config` to contain specific keys

Reply:
I've removed that sentence entirely.